pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -568,12 +568,21 @@ type templateArgs struct {
 	// ServiceAccount is the name of the service account to create in the syncer
 	// namespace on the pcluster.
 	ServiceAccount string
+	// DnsServiceAccount is the name of the DNS service account to create in the syncer
+	// namespace on the pcluster.
+	DnsServiceAccount string
```

Comment:
Global comment: DNS, or dns, but not Dns 😄 (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)